scipy/cluster/vq.py

```diff
@@ -530,28 +544,31 @@ def _krandinit(data, k, rng):
     mu = data.mean(axis=0)
 
     if data.ndim == 1:
-        cov = np.cov(data)
+        cov = xp.cov(data)
```

Comment:
`cov` is not an array API standard function, so this shouldn't work. I guess all tested libraries have it under the same name. It looks like only plain `cov` with no keyword args is used in a bunch of places, which can fairly easily be implemented with standard primitives like `sum`, `mean` and `dot`.